frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -1,30 +1,80 @@
-import { Card } from 'semantic-ui-react';
+import React, { useState, Dispatch, SetStateAction } from 'react';
+import { Button, Card, Form, Icon, Modal, TextArea } from 'semantic-ui-react';
+import ShoutoutsAPI from '../../../API/ShoutoutsAPI';
 import ShoutoutDeleteModal from '../../Modals/ShoutoutDeleteModal';
 import styles from './ShoutoutCard.module.css';
 
-const ShoutoutCard = (props: {
+interface ShoutoutCardProps {
   shoutout: Shoutout;
-  setGivenShoutouts: React.Dispatch<React.SetStateAction<Shoutout[]>>;
-}): JSX.Element => {
-  const { shoutout, setGivenShoutouts } = props;
+  setGivenShoutouts: Dispatch<SetStateAction<Shoutout[]>>;
+}
 
-  const fromString = shoutout.isAnon
-    ? 'From: Anonymous'
-    : `From: ${shoutout.giver?.firstName} ${shoutout.giver?.lastName}`;
-  const dateString = `${new Date(shoutout.timestamp).toDateString()}`;
+const ShoutoutCard: React.FC<ShoutoutCardProps> = ({ shoutout, setGivenShoutouts }) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [editedMessage, setEditedMessage] = useState(shoutout.message);
+
+  const handleEditShoutout = async () => {
+    try {
```

Comment:
I think we can do without this `try/catch` and console statement, since it doesn't do much other than `console.log` the error (which we would get in the console anyways). It would be worth it to try/catch if we had some sort of fallback code we wanted to run or wanted to record metrics.